src/Reducers/ToastReducer.js

```diff
@@ -0,0 +1,43 @@
+const initToastState = {
+    toastList: []
+}
+const toastReducer = (state, { type, payload }) => {
+    console.log(state)
+    console.log(type, payload)
```

Comment:
changes: remove console logs when pushing code to github

Reply:
done